### PR TITLE
Add instructions about upstream builds

### PR DESCRIPTION
**Pull request by jklukas**
Addresses https://github.com/confluentinc/kafka-connect-storage-cloud/issues/94

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,12 @@ Documentation for this connector can be found [here](http://docs.confluent.io/cu
 
 # Development
 
-To build a development version you'll need a recent version of Kafka. You can build
-*kafka-connect-storage-cloud* with Maven using the standard lifecycle phases.
+To build a development version you'll need a recent version of Kafka and you may
```
Comment (kkonstantine) [inline README.md]: How about: 

> To build a development version you'll need a recent version of Kafka as well as a small set of upstream Confluent project, which you'll have to build from their appropriate snapshot branch. See kafka-connect-storage-common FAQ for guidance on this process.

Comment (jklukas) [reply README.md]: Will integrate that.

**Comment by jklukas** [comment]
Added a commit to integrate your suggestion, @kkonstantine

**Comment by kkonstantine** [comment]
ok to test

**Pull request merged**